chat-model/src/main/resources/canon/chat.json

```diff
@@ -1261,6 +1261,29 @@
           }
         }
       },
+      "ReactionDetailsPayload": {
+        "type": "object",
+        "extends": "#/components/schemas/MaestroPayload",
+        "properties": {
+          "action": {
+            "type": "string"
+          },
+          "reactionName": {
+            "type": "string"
+          },
+          "messageId": {
```

Comment:
There is another attribute called messageId in the LiveCurrentMessage containing this structure, I would give it a different name which describes what message this is the ID of subjectMessageId or reactionToMessageId or something like that.

Message IDs should be of type "$ref": "#/components/schemas/MessageId"

Reply:
We would prefer not to rename the property since the model `ReactionDetails` already exists in the `SymphonyLibrary` repository; we're just bringing it here the way it is already defined there.

Reply:
I updated the type